openwisp_firmware_upgrader/tests/test_admin.py

```diff
@@ -618,7 +622,10 @@ def test_upgrade_all(self, *args):
             )
             self.assertEqual(response.status_code, 200)
             self.assertContains(
-                response, '<ul class="errorlist"><li>Enter a valid JSON.</li></ul>'
+                response,
+                ('<ul class="errorlist"{}><li>Enter a valid JSON.</li></ul>').format(
+                    ' id="id_upgrade_options_error"' if django.VERSION >= (5, 2) else ''
+                ),
             )
```

Comment:
Would it be possible to do something like this for better readability?

```
id_attr = 'id="id_upgrade_options_error"' if django.VERSION >= (5, 2) else ''
...
('<ul class="errorlist" {}><li>Enter a valid JSON.</li></ul>').format(id_attr)
```

Reply:
Yes I will modify it